lib_helpers/kafka_consumer.ml

```diff
@@ -132,7 +132,7 @@ let fold_queue
   ?(consumer_props = ["metadata.broker.list","localhost:9092"])
   ?(topic_props = [])
   ?(timeout_ms = 1000)
-  ?(stop_at_end = false)
+  ?stop_at_end:(_stop_at_end = false)
```

Comment:
I think there might be an issue in general there since this argument is never used.

Reply:
The `stop_at_end` argument is used.